docs/multiple_fqdn.md

```diff
@@ -0,0 +1,49 @@
+# Multiple FQDN Support in AMKO using HostRules
+
+AKO provides a CRD called [hostrule](https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/master/docs/crds/hostrule.md) for virtual host properties. These hostrule objects can be utilised by AMKO for mutliple FQDN support and [local to global fqdn mapping](https://github.com/vmware/global-load-balancing-services-for-kubernetes/blob/master/docs/local_and_global_fqdn.md) in AMKO. <br>
+Hostrules can be configured so that a specific application can have the ability to have multiple FQDNs configured under a specific route/ingress for the child VS, instead of having to create the route multiple times.
+
+AMKO can reduce the number of GSLB Services by grouping related(FQDN and its aliases) FQDNs together into one GSLB Service. 
+
+The functionality of AMKO changes based on how AMKO is configured.
```

Comment:
the functionality changes based on how the hostrule is configured?

Reply:
Done